packages/block-editor/src/components/global-styles/style.scss

```
@@ -147,6 +152,14 @@
 	}
 }
 
+.block-editor-global-styles-background-panel__color-tools-panel-item {
+
+	button.components-button {
+
+		padding: 0;
+	}
+}
+
```

Leading/trailing new lines inside a section are not redundant:

```suggestion
.block-editor-global-styles-background-panel__color-tools-panel-item {
	button.components-button {
		padding: 0;
	}
}

```